lib/protocol_9p_client.ml

```diff
@@ -482,6 +483,13 @@ module Make(Log: Protocol_9p_s.LOG)(FLOW: V1_LWT.FLOW) = struct
       | Ok stat -> return (`Ok stat.Types.Stat.length)
       | _ -> return (`Error (Unknown_key key))
 
+    let mem t key =
+      let path = parse_path key in
+      stat t path
+      >>= function
+      | Ok stat -> return (`Ok true)
+      | _ -> return (`Ok false)
```

Comment:
The official 9p-approved way is to match on the error string I believe :-( Would be nice if the 9p client could map strings back to something sensible.